### PR TITLE
[editor] Limit more fields (website, menu, social medias, etc...) for some POI types

### PR DESCRIPTION
**Pull request by Arthur-GYT**
All is in the title ^^.

Related to https://github.com/organicmaps/organicmaps/pull/8052#issuecomment-2118874974

### CODE REVIEW

**Comment by Jean-BaptisteC** [comment]
The right solution is to update editor.config

**Comment by Arthur-GYT** [comment]
@Jean-BaptisteC It's strange. I test it with restaurant and it appear for haircut. Do I miss something ?

File: data/editor.config [hunk 1]
```diff
@@ -354,6 +354,7 @@
       <type id="amenity-restaurant" group="food">
         <include group="poi_internet" />
         <include field="cuisine" />
+        <include field="website_menu" />
```
Comment (biodranik) [inline data/editor.config:357]: Please review all related POIs at least in group=food, and also add missing cuisine tags if necessary.

Why was it displayed for Parking?

Comment (Arthur-GYT) [reply data/editor.config:357]: I forgot to add the field in details pane

Comment (Arthur-GYT) [reply data/editor.config:361]: I think it's done. Also, I don't see missing cuisine tags.

**Comment by Arthur-GYT** [comment]
@biodranik  I've changed the scope of the PR. I've also fixed the fact that, for example, parking lots were displaying fields for social medias in the editor (which doesn't make sense).

File: android/app/src/main/java/app/organicmaps/editor/EditorFragment.java [hunk 2]
```diff
@@ -469,6 +469,16 @@ private void initViews(View view)
     mDetailsBlocks.put(Metadata.MetadataType.FMD_PHONE_NUMBER, blockPhone);
     mDetailsBlocks.put(Metadata.MetadataType.FMD_CUISINE, blockCuisine);
     mDetailsBlocks.put(Metadata.MetadataType.FMD_INTERNET, blockWifi);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE, websiteBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE_MENU, websiteMenuBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE_MENU, websiteMenuBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_EMAIL, emailBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_CONTACT_FACEBOOK, facebookContactBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_CONTACT_INSTAGRAM, instagramContactBlock);
```
Comment (biodranik) [inline android/app/src/main/java/app/organicmaps/editor/EditorFragment.java:477]: How did it work before?

File: android/app/src/main/java/app/organicmaps/editor/EditorFragment.java [hunk 3]
```diff
@@ -469,6 +469,16 @@ private void initViews(View view)
     mDetailsBlocks.put(Metadata.MetadataType.FMD_PHONE_NUMBER, blockPhone);
     mDetailsBlocks.put(Metadata.MetadataType.FMD_CUISINE, blockCuisine);
     mDetailsBlocks.put(Metadata.MetadataType.FMD_INTERNET, blockWifi);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE, websiteBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE_MENU, websiteMenuBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE_MENU, websiteMenuBlock);
```
Comment (biodranik) [inline android/app/src/main/java/app/organicmaps/editor/EditorFragment.java:474]: Copy-paste from above line?

File: data/editor.config [hunk 4]
```diff
@@ -1051,6 +1056,7 @@
       </type>
       <type id="amenity-biergarten" group="food">
         <include group="poi_internet" />
+        <include field="website_menu" />
```
Comment (biodranik) [inline data/editor.config:1059]: Does it make sense to allow editing cuisine for a biergarten? @organicmaps/translations-de WDYT?

File: android/app/src/main/java/app/organicmaps/editor/EditorFragment.java (hunk 2 above)
Comment (Arthur-GYT) [reply android/app/src/main/java/app/organicmaps/editor/EditorFragment.java:477]: Previously, it ignored the configuration described in editor.config. You need to add the field as shown here to make it take this configuration into account.

File: android/app/src/main/java/app/organicmaps/editor/EditorFragment.java (hunk 3 above)
Comment (Arthur-GYT) [reply android/app/src/main/java/app/organicmaps/editor/EditorFragment.java]: Fixed ^^.

File: data/editor.config (hunk 4 above)
Comment (Arthur-GYT) [reply data/editor.config:1060]: According to the wiki, it make sense ^^ 

https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbiergarten

File: android/app/src/main/java/app/organicmaps/editor/EditorFragment.java (hunk 2 above)
Comment (Arthur-GYT) [reply android/app/src/main/java/app/organicmaps/editor/EditorFragment.java:476]: @biodranik Do I need to change anything?

**Review by biodranik — APPROVED** [review]
LGTM, Did you test it? Note that debug build of OM uploads data into the dev server, and you need a separate account there:

https://master.apis.dev.openstreetmap.org/

If you need to edit some data from the main OSM server, you can upload it to the dev server using this tool: https://github.com/Zverik/osm_to_sandbox

@Jean-BaptisteC PTAL if you have time.

**Review by Jean-BaptisteC — APPROVED** [review]
✅ Pixel 6 - Android 14
https://master.apis.dev.openstreetmap.org/node/4349085700

**Pull request merged**